templates/README.md

```diff
@@ -0,0 +1,68 @@
+Topic name
+==========
+
+*Short introduction about the topic, why it is important to learn it (the questions behind this topic), what are the objectives, ...*
+
```

Comment:
Should this go into the template with defined categories, like `1d`, `3h`, `6h`?